tests/ttnn/unit_tests/operations/test_moreh_linear.py

```diff
@@ -16,24 +15,129 @@
 )
 
 
-def moreh_linear(shapes, has_bias, has_output, compute_kernel_config, device):
+def create_ttnn_tilized_tensor(torch_tensor, device, dtype):
```

Comment:
I can see that this function is only being used exactly once in `get_tensors`. I don’t think you should create a function to be used only once, since it basically just clutters the code.

You could either just integrate this directly, or move your `ttnn.from_torch` calls to this instead, whichever way you choose

Reply:
Resolved in latest commit